app.js

```diff
@@ -25,9 +25,10 @@ const logRequest = (req, res, next) => {
     next();
```

Comment:
Remove the next parameter instead

Reply:
Don't move where the logReq function is called.

Reply:
I am sorry, the first change is correct. Readd the next parameter and use the function as middleware again.